roles/openshift_health_checker/openshift_checks/logging/fluentd_config.py

```diff
@@ -0,0 +1,138 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+from openshift_checks import OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+class FluentdConfig(LoggingCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    def is_active(self):
+        logging_deployed = self.get_var("openshift_hosted_logging_deploy", default=False)
+
+        try:
+            version = self.get_major_minor_version(self.get_var("openshift_image_tag"))
```

Comment:
Note that this can raise `OpenShiftCheckException`. We should protect the action plugin call to `is_active` with a `try... except` block for cases like this, to prevent the action plugin run from blowing up. I can include that in my current work branch.